pipelines/LSSTComCam/quickLook.yaml

```diff
@@ -24,6 +24,9 @@ tasks:
   calibrate:
     class: lsst.pipe.tasks.calibrate.CalibrateTask
     config:
+      requirePhotoCal: false
+      requireAstrometry: false
+      astrometry.matcher.maxOffsetPix: 1500
```

Comment:
Yep, the matcher.maxOffsetPix is already default here: https://github.com/lsst/obs_lsst/blob/main/config/comCam/calibrate.py

so remove: `astrometry.matcher.maxOffsetPix: 1500`